kinto/config/kinto.tpl

```diff
@@ -78,9 +159,80 @@ multiauth.policies = basicauth
 # fxa-oauth.relier.enabled = true
 # fxa-oauth.webapp.authorized_domains = *
```

Comment:
I think we could get rid of Firefox Accounts since its usage on any server is not opened to the public.